docs/shooter_tutorial.html

```diff
@@ -263,6 +263,7 @@ <h3 id="whats-next">7: What’s next?</h3>
 <li><p>Better UI and main menu.</p></li>
 <li><p>Game structure: different levels the player can try to beat, permanent upgrades the player can buy, etc.</p></li>
 </ul>
+<p>Also, at this point, our game has no sound at all. If you'd like to see how to add some anbient or enemy sounds, have a look at the use of the <code>play_</code> and <code>sound_</code> functions in <code>tut_sound.lobster</code>. This example also shows how to to dynamically adjust the volume and pause or resume individual sounds.</p>
```

Comment:
you'll want to change this in the .md, it will get overwritten otherwise

Reply:
Ah -- wasn't even aware of the source files. Done.